package.json

```diff
@@ -39,7 +39,7 @@
     "@dappnode/schemas": "^0.1.20",
     "@dappnode/toolkit": "^0.1.21",
     "@dappnode/types": "^0.1.39",
-    "@octokit/rest": "^18.0.12",
+    "@octokit/rest": "^20.1.1",
```

Comment:
I would enforce the sdk to run in node >= 18 according to official docs, see https://github.com/octokit/octokit.js?tab=readme-ov-file#fetch-missing

```json
  "engines" : { 
    "node" : ">=18.0.0"
  }
```

Reply:
This is already included in `master`